SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -59,12 +59,12 @@ USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& O
 	, bBuildAndUploadAssembly(true)
 	, AssemblyBuildConfiguration(TEXT("Development"))
 	, bConnectServerToCloud(false)
-	, LocalReceptionistPort(SpatialConstants::DEFAULT_SERVER_RECEPTIONIST_PROXY_PORT)
 	, ListeningAddress(SpatialConstants::LOCAL_HOST)
 	, SimulatedPlayerDeploymentRegionCode(ERegionCode::US)
 	, bPackageMobileCommandLineArgs(true)
 	, bStartPIEClientsWithLocalLaunchOnDevice(false)
 	, SpatialOSNetFlowType(ESpatialOSNetFlow::LocalDeployment)
+	, LevelEditorPlaySettings(GetDefault<ULevelEditorPlaySettings>())
```

Comment:
I don't think we want to keep a reference to this, instead we should just always call `GetDefault<ULevelEditorPlaySettings>` when we want to use it. This is the pattern elsewhere in the engine I believe.

Reply:
Hmm, I just thought of storing this reference as a sort of performance boost, is there any particular drawback of using this method?